zeppelin-zengine/src/main/java/org/apache/zeppelin/plugin/PluginManager.java

```diff
@@ -60,7 +60,8 @@ public NotebookRepo loadNotebookRepo(String notebookRepoClassName) throws IOExce
             (Class.forName(notebookRepoClassName).newInstance());
         return notebookRepo;
       } catch (InstantiationException | IllegalAccessException | ClassNotFoundException e) {
-        LOGGER.warn("Fail to instantiate notebookrepo from classpath directly:" + notebookRepoClassName, e);
+        LOGGER.warn("Fail to instantiate notebookrepo from classpath directly:"
+            + notebookRepoClassName);
```

Comment:
think useful to keep the exception in the log though?

Reply:
Fixed